Dockerfile.django

```diff
@@ -71,12 +71,17 @@ COPY \
   /
 COPY wsgi.py manage.py tests/unit-tests.sh ./
 COPY dojo/ ./dojo/
+
+# Add extra fixtures to docker image which are loaded by the initializer
+COPY docker/extra_fixtures/* /app/dojo/fixtures/
+
 # Legacy installs need the modified settings.py, do not remove!
 RUN \
   cp dojo/settings/settings.dist.py dojo/settings/settings.py
 COPY tests/ ./tests/
 RUN \
   rm -f /readme.txt && \
+  rm -f dojo/fixtures/readme.txt && \
```

Comment:
What do you think of adding this file to .dockerignore instead of deleting it afterwards?

Reply:
Thanks, forgot about this one!

I'll ignore /readme.txt coming from docker/certs/readme.txt as well.

Reply:
@alles-klar I revert the change. If I add readme.txt in dockerignore, COPY will fail because the directory is empty.

Reply:
can we put a comment in there so everybody knows why this is happening|?

Reply:
Yep, it's done